datahub-web-react/src/app/ingest/source/builder/RecipeForm/common.tsx

```diff
@@ -127,12 +129,14 @@ const dashboardAllowFieldPath = 'source.config.dashboard_pattern.allow';
 export const DASHBOARD_ALLOW: RecipeField = {
     name: 'dashboard_pattern.allow',
     label: 'Allow Patterns',
-    tooltip: 'Use regex here.',
+    tooltip:
+        'Only include specific Dashboards by providing the name of a Dashboard, or a Regular Expresssion (REGEX). If not provided, all Dashboards will be included.',
```

Comment:
Looker Ingestion source filters by dashboard id (since thats unique) and not name.